### PR TITLE
[docs] Revise integrated development tips

### PR DESCRIPTION
**Pull request by niedzielski**
Revise tips for integrated development workflows, add some Input JSDocs,
and remove a test-only function.

### CODE REVIEW

File: src/components/input/Input.vue [hunk 1]
```diff
@@ -78,6 +78,10 @@ export const mwIconInfo = 'M9.5 16A6.61 6.61 0 0 1 3 9.5 6.61 6.61 0 0 1 9.5 3 6
 
 export default Vue.extend( {
 	name: 'WvuiInput',
+	/**
+	 * All atributes set on the components such as disabled and type are passed to the underlying
```
Comment (berndsi) [inline src/components/input/Input.vue:82]: typo.

BTW, I noticed another one below: `properyy` on line 110. Not sure if it's worth its own PR.

File: src/components/input/Input.vue [hunk 2]
```diff
@@ -123,7 +128,7 @@ export default Vue.extend( {
 				'wvui-input--clearable': this.clearable
 			};
 		},
-		iconName(): string {
+		iconName(): string|undefined {
```
Comment (berndsi) [inline src/components/input/Input.vue:131]: Should there be a unit test associated with this return type fix?

Comment (niedzielski) [reply src/components/input/Input.vue]: It looks like this method was only used by tests. I've opted to delete the method instead.

Comment (berndsi) [reply src/components/input/Input.vue]: Even better. :)

**Comment by berndsi** [comment]
@niedzielski Did you want to update the PR message before I squash and merge it? The "fix a return type" turned into something else.

**Review by speedandfunction-anribolon — APPROVED** [review]

**Comment by niedzielski** [comment]
> The "fix a return type" turned into something else.

Revised :+1:

**Pull request merged**